lattices/LRegions/LCSlicer.cc

```diff
@@ -327,53 +327,55 @@ void LCSlicer::fill()
         itsFracBlc.resize (nr, True);
         itsAbsRelBlc.resize (nr, True);
 	for (uInt i=nrblc; i<nr; i++) {
-	    itsBlc(i) = Slicer::MimicSource;
-	    itsFracBlc(i) = False;
-	    itsAbsRelBlc(i) = RegionType::Abs;
+	    itsBlc[i] = Slicer::MimicSource;
+	    itsFracBlc[i] = False;
+	    itsAbsRelBlc[i] = RegionType::Abs;
 	}
     }
     if (nrtrc < nr) {
         itsTrc.resize (nr, True);
         itsFracTrc.resize (nr, True);
         itsAbsRelTrc.resize (nr, True);
 	for (uInt i=nrtrc; i<nr; i++) {
-	    itsTrc(i) = Slicer::MimicSource;
-	    itsFracTrc(i) = False;
-	    itsAbsRelTrc(i) = RegionType::Abs;
+	    itsTrc[i] = Slicer::MimicSource;
+	    itsFracTrc[i] = False;
+	    itsAbsRelTrc[i] = RegionType::Abs;
 	}
     }
     if (nrinc < nr) {
         itsInc.resize (nr, True);
         itsFracInc.resize (nr, True);
 	for (uInt i=nrinc; i<nr; i++) {
-	    itsInc(i) = 1;
-	    itsFracInc(i) = False;
+	    itsInc[i] = 1;
+	    itsFracInc[i] = False;
 	}
     }
     // Check if the region is absloute, fractional, unspecified, or strided.
+    // The float representation of MimicSource is inaccurate (648 instead of 646
+    // as the last digits), so be relaxed in testing it.
     itsIsFractional  = False;
     itsIsAbsolute    = True;
     itsIsUnspecified = False;
     itsIsStrided     = False;
     for (uInt i=0; i<nr; i++) {
-        if (itsBlc(i) == Slicer::MimicSource) {
+        if (static_cast<Int64>(itsBlc[i]) < Slicer::MimicSource+10) {
```

Comment:
I have no idea what `Slicer::MimicSource + 10` means, could you explain? Or maybe someone more familiar with lattices could have a look?